curriculum/challenges/english/25-front-end-development/lab-sum-all-numbers-algorithm/6827cc540b8818ebf41d581d.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 6827cc540b8818ebf41d581d
+title: Design a Sum All Numbers Algorithm
+challengeType: 26
+dashedName: lab-sum-all-numbers-algorithm
+---
+
+# --description--
+
+In this lab, you will need to design a sum all numbers algorithm.
+
+Your `sumAll` function accepts an array of two numbers. Your function should return the sum of those two numbers plus the sum of all the numbers between them. The lowest number will not always come first.
+
+For example, `sumAll([4,1])` should return `10` because sum of all the numbers between 1 and 4 (both inclusive) is `10`.
```

Comment:
```suggestion
1. You should have a function named `sumAll` function that accepts an array of two numbers.
1. `sumAll([n, m])` should return the sum `n` and `m` plus the sum of all the numbers between them. The lowest number will not always come first. For example, `sumAll([4,1])` should return `10` because sum of all the numbers between `1` and `4` (both inclusive) is `10`.
```